docs/server/installation.rst

```diff
@@ -107,7 +107,7 @@ To verify that everything installed correctly, you should be able to access the
 .. code-block:: bash
 
   (env) $ pootle --version
-  Pootle 2.7.1 (Django 1.7.10, Translate Toolkit 1.13.0)
+  Pootle 2.7.2 (Django 1.7.10, Translate Toolkit 1.13.0)
```

Comment:
This change rather belongs to docs adjustment when 2.7.2 release is being done, rather to a general docs cleanup, isn't it?

I prefer this specific change to be moved outside of this PR, so these docs cleanups can be cherry picked to the stable/2.7.1 branch.